src/index.js

```diff
@@ -43,9 +43,8 @@ const startApp = () => {
 };
 
 if (isElectron()) {
-  const { webFrame, ipcRenderer } = require('electron'); // eslint-disable-line global-require
+  const { webFrame, ipcRenderer } = window.require('electron'); // eslint-disable-line global-require
```

Comment:
I'm a bit confused about the re-introduction of `window.require` - didn't we work around this here: https://github.com/codaco/Network-Canvas/pull/447 ?

If so, what has changed?